en/08_Changelogs/6.0.0.md

```diff
@@ -33,6 +33,9 @@ CanonicalURLMiddleware::singleton()->setEnabledEnvs([
 
 ### Other new features
 
+- Native indexed PHP arrays can now be passed into templates and iterated over with `<% loop $MyArray %>`. Under the hood they are wrapped in [`ArrayList`](api:SilverStripe\View\ViewableData), so you can get the count using `$Count` and use `<% if $ArrayList %>` as a shortcut for `<% if $ArrayList.Count %>`. Other functionality from `ArrayList` such as filtering and sorting cannot be used on arrays since they don't have keys to filter or sort against.
+  - Note that associative arrays will continue to be wrapped in `ArrayData`, which means you can access their values by key but may not be able to iterate over them.
```

Comment:
```suggestion
```

Reply:
They are still wrapped if they're nested in an array/ArrayList - but you can't return them directly from a controller.  I'm not sure if it's even worth mentioning assoc arrays here because their usage is inconsistent and confusing

Reply:
Not worth mentioning, since the work to deal with those was split out into https://github.com/silverstripe/silverstripe-framework/issues/11262

Reply:
Done